packages/server/lib/utils/asyncWrapper.ts

```diff
@@ -13,7 +14,15 @@ export function asyncWrapper<TEndpoint extends Endpoint<any>, Locals extends Rec
 ): RequestHandler<any, TEndpoint['Reply'], any, any, any> {
     return (req, res, next) => {
         const active = tracer.scope().active();
-        active?.setTag('http.route', req.route?.path || req.originalUrl);
+        if (active) {
```

Comment:
there is a hooks attributes for the ddtrace express plugin https://datadoghq.dev/dd-trace-js/interfaces/export_.plugins.express.html#hooks 
could we use this instead?

Reply:
it's handy to have it here (or at least in the other code) because it's shared across multiple services, otherwise, I need to copy the same code in each `tracer.ts`. Wdyt?

Reply:
do we need it for all services though?
Up to you